modules/text/include/opencv2/text/erfilter.hpp

```diff
@@ -227,6 +227,43 @@ features: hole area ratio, convex hull ratio, and number of outer inflexion poin
 CV_EXPORTS_W Ptr<ERFilter> createERFilterNM2(const Ptr<ERFilter::Callback>& cb,
                                                   float minProbability = (float)0.3);
 
+/** @brief Create an Extremal Region Filter for the 1st stage classifier of N&M algorithm [Neumann12].
+
+@param  filename :   The path to the classifier XML, e.g. /path/to/cpp/trained_classifierNM1.xml
+@param  thresholdDelta :   Threshold step in subsequent thresholds when extracting the component tree
+@param  minArea :   The minimum area (% of image size) allowed for retreived ER's
+@param  maxArea :   The maximum area (% of image size) allowed for retreived ER's
+@param  minProbability :   The minimum probability P(er|character) allowed for retreived ER's
+@param  nonMaxSuppression :   Whenever non-maximum suppression is done over the branch probabilities
+@param  minProbabilityDiff :   The minimum probability difference between local maxima and local minima ERs
+
+The component tree of the image is extracted by a threshold increased step by step from 0 to 255,
```

Comment:
@abratchik  Here can be something like this instead of the long description.
```
/** @brief Reads an Extremal Region Filter for the 1st stage classifier of N&M algorithm
    from the provided path e.g. /path/to/cpp/trained_classifierNM1.xml
@overload
 */
```